src/core/query-builder.utils.test.ts

```diff
@@ -0,0 +1,111 @@
+import { expressionBuilderCallback, expressionReaderCallback, setQueryComputedFields } from "./query-builder.utils"
+
+jest.mock('./query-builder.constants.ts', () => ({
+  ...jest.requireActual('./query-builder.constants'),
+  ComputedDataFields: {
+    Obj1: 'obj1.prop1 || obj1.prop2',
+    Obj2: 'obj2.prop1 || obj2.extra.prop2 || obj2.prop3 || obj2.prop4 || obj2.prop5 || obj2.prop6',
+    Obj3: 'obj3.prop1 || obj3.prop2 || obj3.prop3'
+  }
+}));
+
+describe('QueryBuilderUtils', () => {
+  describe('setQueryComputedFields', () => {
+    test('should not modify the query if there are no computed fields', () => {
+      const query = 'SELECT * FROM table WHERE field1 = "value1" AND field2 = "value2"';
```

Comment:
I'm confused by the usage of this SQL query. Can we change it to something that's more relevant ?